et_replay/comm/backend/pytorch_dist_backend.py

```diff
@@ -1084,11 +1101,9 @@ def initialize_groups(self, backend="gloo"):
                 pg = self.get_default_group()
             else:
                 pg = self.get_new_pg(group_ranks=group_ranks, backend=backend)
-                global_rank = self.get_global_rank()
-                if global_rank in group_ranks:
-                    logger.info(
-                        f"initialize_groups: Rank {global_rank} creates new group pg_id {pg_id} {pg} with {group_ranks}"
-                    )
+                logger.info(
+                    f"initialized_group: create new group pg_id {pg_id} {pg} with {group_ranks}"
+                )
```

Comment:
Is the following msg better:

f"initialized_group: create new group: pg_id = {pg_id}  ranks = {group_ranks}"

Reply:
Sure, you can update it.

Reply:
if use var and value, I prefer to use comma for separation like this, `f"initialized_group: create new group, pg_id = {pg_id}, ranks = {group_ranks}"`